frontend/src/pages/modelRegistry/screens/RegisterModel/useRegisterModelData.ts

```diff
@@ -47,6 +54,13 @@ const registrationCommonFormDataDefaults: RegistrationCommonFormData = {
   modelLocationURI: '',
   modelCustomProperties: {},
   versionCustomProperties: {},
+  modelSourceGroup: '',
+  modelSourceId: '',
+  modelSourceName: '',
+  catalogSourceName: '',
+  catalogRepositoryName: '',
+  catalogModelName: '',
+  catalogModelTag: '',
```

Comment:
We can remove these - anything you add to `RegistrationCommonFormData` is inherited here.